docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  smf:
+    build: .
+    ports:
+      - "8080:80"
+    volumes:
+      - ./:/var/www/html
+  database:
+    image: mysql
+    restart: always
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: password
```

Comment:
That's secure looking!

Reply:
I think you're missing the point.   This is meant for development, nothing else.

If we want to make this ready for production, someone else with more docker experience can take this over.